cli/dcoscli/data/help/node.txt

```diff
@@ -23,7 +23,8 @@ Options:
                             configuration, the private slaves are unreachable from the public
                             internet. You can access them using this option, which will first hop
                             from the publicly available master.
-    --slave=<slave-id>      Access the slave with the provided ID
+    --slave=<slave-id>      Deprecated. Please use --id.
+    --id=<node-id>          Access the node with the provided ID
```

Comment:
And mentioning `SSH` in the arg's description doesn't make sense, since it also applies to `node log`.